libs/oeth/history/src/routes.tsx

```diff
@@ -0,0 +1,30 @@
+import { defineMessage } from 'react-intl';
+
+import { HistoryView } from './views/HistoryView';
+import { HistoryViewOETH } from './views/HistoryViewOETH';
+import { HistoryViewWOETH } from './views/HistoryViewWOETH';
+
+import type { NonIndexRouteObject } from 'react-router-dom';
+
+export const historyRoute: NonIndexRouteObject = {
+  path: '/history',
+  Component: HistoryView,
+  handle: { label: defineMessage({ defaultMessage: 'History' }) },
+  children: [
+    {
+      index: true,
+      Component: HistoryViewOETH,
+      handle: { label: defineMessage({ defaultMessage: 'OETH' }) },
+    },
+    {
```

Comment:
I think you can remove that route since it's not used

Reply:
feels weird not having it 🤣

Reply:
doesn't hurt to have it.